functions.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+function displayPDO() {
+    $db = new PDO('mysql:dbname=portfolio;host=127.0.0.1', 'root');
+    $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    return $db;
+}
+
+displayPDO();
```

Comment:
this function returns the $db, but your not putting it anywhere. Did you mean to call this here?

Reply:
I have since removed it and everything works.